### PR TITLE
chore/improve readme and ignore local config file

### PR DESCRIPTION
**Pull request by sebbi08**

### CODE REVIEW

File: README_dev.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ npm run test:local -- testSuiteName
           "platformClientId": "...",
           "platformClientSecret": "..."
       },
-      "database": { "driver": "lokijs", "folder": "./" },
+      "database": { "driver": "lokijs", "folder": "./", "dbName": "local" },
```
Comment (jkoenig134) [inline README_dev.md:109]: why? we have a default for that.

Comment (sebbi08) [reply README_dev.md:109]: nop for me, it was not working without that config the default is only in the docker compose file I suppose

Comment (jkoenig134) [reply README_dev.md:109]: image
we have a default, we have to check elsewhere why it doesn't pulls this default for you.

Comment (jkoenig134) [reply README_dev.md:109]: (but for now, please remove it from here)

**Review by jkoenig134 — APPROVED** [review]

**Pull request merged**